charts/enterprise-logs/Chart.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: "v1"
+name: enterprise-logs
+version: 0.1.0
+appVersion: v2.2.1
+kubeVersion: "^1.10.0-0"
+description: "Grafana Enterprise Logs"
+home: https://grafana.com/products/enterprise/logs/
+icon: tbd
```

We don't yet have a GEL logo so we can use the Loki one

```suggestion
icon: https://grafana.com/docs/loki/latest/logo_and_name.png
```